rfcs/table.md

```diff
@@ -0,0 +1,321 @@
+         );
+    }
+}
+```
+
+
+## Exports
+
+* `Table`
+* `StatefulTable`
+* `Tbody`
+* `Thead`
+* `TheadTr`
+* `Td`
+* `Th`
+* `Tr`
+
+## `Table` API
+
+* `data:Array<T>` - Required
+  Description of prop
+* `columns: Column` - Required
+  Description of prop
+* `render: (dataSource, columns) => $ReactNode` - Optional
+    Custom render table body.
+* `overrides - Object:` - Optional
+
+* `overrides(continued):`
+  * `Tbody` - Body of the table. Corresponds.
+  * `Thead` - Container for the table header related elements.
+  * `TheadTr` - A header row. `Tr` for the `Thead` element.
+  * `Th` - A header cell. Corresponds with the `th` html element.
+  * `Td` - A data cell in the table. Corresponds with the `td` html element.
+  * `Tr` - A data row that contains `Td` components.
+
+
+## `Column` API
+One of the Table columns prop for describing the table's columns, Column has the same API.
+* `title: string` - Required. Title of the column to display in the header.
+* `dataIndex: string` - Optional. Attribute at which to index the row data.
+* `render: <T>(<T>, index, Array<T>) => $ReactNode` - Optional.	Renderer of the table cell. The return value should be a ReactNode.
+* `sorter:Function`- Optional  Sort function for [local sort](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort)
+* `defaultSortOrder:'ascend'|'descend'`: Only to be declared on one of the columns, this will be define the default sort on the initial set of data.
+
+## Presentation Components
+
+### `Tbody` API
+Please refer to the [tbody DOM API](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/tbody)
+
+### `Thead` API
+Please refer to the [thead DOM API](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/thead)
+
+### `TheadTr` API
+Please refer to the [tr DOM API](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/tr)
+
+### `Td` API
+Please refer to the [td DOM API](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/td)
+
+### `Th` API
+Please refer to the [th DOM API](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/th)
+
+### `Tr` API
+Please refer to the [th DOM API](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/tr)
+
+
+### Dependencies
+
+Does this component depend on any 3rd party packages or other internal components?
+
+## Accessibility
+
+How can this component be used via keyboard controls?
+No keyboard shortcuts will be enabled.
+
+What are the accessibility best practices for this component (aria-\*, role, etc.)
+
+We are making the assuming that in most use cases, a [table with one header](https://www.w3.org/WAI/tutorials/tables/one-header/)
+will be rendered.
+
+The [W3 guideline for table states](https://www.w3.org/WAI/tutorials/tables/):
+> Header cells must be marked up with `<th>`, and data cells with `<td>` to make tables accessible. For more complex tables, explicit associations may be needed using `scope`, `id`, and `headers` attributes.
```

Comment:
Ahhh...this is certainly a benefit of sticking to the HTML table spec...hm. I do see ARIA roles on `react-virtualized`, at least: https://github.com/bvaughn/react-virtualized/issues/606

Reply:
react-virtualized does add aria-* roles & sets tabIndex to assist with keyboard nav. 
It also provides some additional components ie: ArrowKeyStepper to help with navigating virtualized lists